File: beginner/pagina02/1047/1047.cs
using System;
using System.Linq;

public class URI
{
    static void Main()
    {
        int[] input = Console.ReadLine().Split(' ').Select(int.Parse).ToArray();
        int h1 = input[0], m1 = input[1], h2 = input[2], m2 = input[3];

        int inicio = h1 * 60 + m1;
        int fim = h2 * 60 + m2;

        int duracao = fim - inicio;
        if (duracao <= 0)
        {
            duracao += 24 * 60; // Adiciona 24h se terminou no dia seguinte ou mesmo horário
        }

        int horas = duracao / 60;
        int minutos = duracao % 60;

        Console.WriteLine($"O JOGO DUROU {horas} HORA(S) E {minutos} MINUTO(S)");
    }
}

File: beginner/__1000__/1047/1047.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);

        int h1 = in.nextInt();
        int m1 = in.nextInt();
        int h2 = in.nextInt();
        int m2 = in.nextInt();

        int hr, min;

        if(h2 < h1 || (h2 == h1 && m2 < m1)){
            h2 += 24;
        }

        if(m2 < m1){
            m2 += 60;
            h2 -= 1;
        }

        hr = h2 - h1;
        min = m2 - m1;

        if(hr == 0 && min == 0){
            hr = 24;
            min = 0;
        }

        System.out.println("O JOGO DUROU "  + hr + " HORA(S) E "  + min + " MINUTO(S)");

        in.close();
    }
}